templates/synapse.service.j2

```diff
@@ -0,0 +1,17 @@
+# {{ ansible_managed }}
+[Unit]
+Description=Matrix Synapse homeserver
+
+[Service]
+Type=notify
+NotifyAccess=main
+User={{ synapse_user }}
+ExecStart={{ synapse_fact_virtualenv_dir }}/bin/python -m synapse.app.homeserver --config-path={{ synapse_data_dir }}/homeserver.yaml
```

Comment:
not `python3`?

Reply:
No, there's only one python version in a virtualenv.